src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
```

Comment:
`days * 40` looks like a new var, as well as 20, consider applying to the rest of your value (the checklist, p.1)